lib/canvas_customize_passwords/engine.rb

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2022 Atomic Jolt
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+module CanvasCustomizePasswords
+  NAME = "Canvas Customize Passwords".freeze
+  DISPLAY_NAME = "Enables custom password requirements".freeze
+  DESCRIPTION = "Enables custom password requirements".freeze
+
+  class Engine < ::Rails::Engine
+    config.paths["lib"].eager_load!
+
+    config.to_prepare do
+      Canvas::Plugin.register(
+        :canvas_customize_passwords,
+        nil,
+        name: -> { I18n.t(:canvas_fips_name, NAME) },
```

Comment:
Copy paste error here and the next line with `canvas_fips_name` I think

Reply:
dang it.

Reply:
fixed